contracts/mock/yearn/MockYearnVault.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.10;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import {IYearnVault} from "../../interfaces/yearn/IYearnVault.sol";
+
+contract MockYearnVault is IYearnVault, ERC20 {
+    using SafeERC20 for IERC20;
+
+    IERC20 immutable underlying;
+
+    constructor(
+        string memory _name,
+        string memory _symbol,
+        address _underlying
+    ) ERC20(_name, _symbol) {
+        underlying = IERC20(_underlying);
+    }
+
+    function deposit(uint256 amount, address recipient)
+        public
+        returns (uint256)
+    {
+        require(amount > 0);
+        underlying.safeTransferFrom(msg.sender, address(this), amount);
+        return _issueSharesForAmount(recipient, amount);
+    }
+
+    function pricePerShare() public view returns (uint256) {
+        uint256 totalSupply = totalSupply();
+        if (totalSupply == 0) return 1e18;
+        return (1e18 * _freeFunds()) / totalSupply;
+    }
+
+    function withdraw(
+        uint256 maxShares,
+        address recipient,
+        uint256 maxLoss
```

Comment:
`maxLoss` parameter is unused and causes a compiler warning
`Warning: Unused function parameter. Remove or comment out the variable name to silence this warning.
  --> contracts/mock/yearn/MockYearnVault.sol:41:9:
   |
41 |         uint256 maxLoss
   |`

Reply:
hummm, honestly I would rather keep the parameter just to I know what the variable is, can't we just ignore this somehow? This is a mock so it doesn't really matter

Reply:
fixed with a comment